geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/connection/AuthIntegrationTest.java

```diff
@@ -53,22 +54,33 @@ public int getPort() {
     return port;
   }
 
+  @Override
+  public String getUsername() {
+    return "dataWrite";
+  }
+
+  @Override
+  public String getPassword() {
+    return "dataWrite";
+  }
+
   public void setupCacheWithSecurity() throws Exception {
-    setupCache(true, true);
+    setupCache(getUsername(), true);
   }
 
   public void setupCacheWithoutSecurity() throws Exception {
-    setupCache(false, false);
+    setupCache(null, false);
   }
 
-  private void setupCache(boolean withUsername, boolean withSecurityManager) throws Exception {
+  private void setupCache(String username, boolean withSecurityManager) throws Exception {
+    System.setProperty("io.netty.eventLoopThreads", "1");
```

Comment:
Authentication and authorization is sometimes implemented and handled using thread locals. Our security implementation can do this but the way it's used here does not (and must not) since netty threads are shared between connections. By using a single thread we can be sure that tests using multiple connections will not inadvertently leak  auth data across connections.
I've added some similar comments into the code.

Reply:
Thanks for the explanation!